.github/workflows/cache-cleanup.yml

```diff
@@ -32,4 +32,4 @@ jobs:
         env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           REPO: ${{ github.repository }}
-          BRANCH: "refs/pull/${{ github.event.pull_request.number }}/merge"
+          BRANCH: ${{ "refs/pull/${{ github.event.pull_request.number }}/merge" }}
```

Comment:
Is that needed? It shouldn't be. You might need to just remove the quotes

Reply:
https://github.com/expo/expo-github-action/blob/7bc946c5f6b768b1a1858c1722b3aac4e0064c23/command/README.md?plain=1#L75

Looks like not.